components/browser_kit.rst

```diff
@@ -112,6 +112,28 @@ provides access to the link properties (e.g. ``$link->getMethod()``,
     $link = $crawler->selectLink('Go elsewhere...')->link();
     $client->click($link);
 
+The :method:`Symfony\\Component\\BrowserKit\\AbstractBrowser::click` and
+:method:`Symfony\\Component\\BrowserKit\\AbstractBrowser::clickLink` methods
+can take a second, optional parameter, which is ``serverParameters``. This
```

Comment:
```suggestion
can take optional ``serverParameters``. This
```
Easier to read ?

Reply:
Yes, I rephrased! :slightly_smiling_face: